### DISCUSSION TITLE
About the choice of loss_weights

### DISCUSSION THREAD

**Discussion by ZSTanone** [lululxvi/deepxde#1008] [Q&A]
Hi Dr.Lu and deepxde users,
I've recently been working on the inverse problem of the odes to estimate parameters. I find set different **loss_weights** will lead to changes in results.
![image](https://user-images.githubusercontent.com/103397565/200119117-fe32180f-eab5-40dc-a9ff-ff87e8d368a5.png)
![image](https://user-images.githubusercontent.com/103397565/200118537-03619de2-aacd-4ef6-805a-5975a7050260.png)

We know that in solving some stiffness equations，their solutions will differ by many orders of magnitude，so we can add output_transform to fix that .However,in this case, is there a conventional way to set the loss_weights? A common idea is to keep the loss of the initialized model on the same order of magnitude.And there are also proposals that set the loss_weights of obsevers bigger than odes in inverse problem ,Does this make any sense？

**Comment by lululxvi** [comment]
Yes, it makes sense.
